plugins/services/services_inspector.rb

```diff
@@ -92,10 +92,22 @@ def inspect_sysvinit_services
     services.sort_by(&:name)
   end
 
-  def inspect_upstart_services
-    output = @system.run_command("/sbin/initctl", "show-config", "-e", stdout: :capture)
+  def inspect_ubuntu_services
+    # Ubuntu is managing its services using upstart but also has some
+    # services still not handled by that. Therefor we need to scan upstart
+    # and sysV to determine all services and their bootup state
+    services = parse_ubuntu_upstart
+    parse_ubuntu_sysv.each do |service|
```

Comment:
The `sysv` services are part of the same list as the `upstart` ones and the list has the `init_system` set to ` upstart`. Is that still correct or do we need two lists instead?